networkit/cpp/io/test/IOGTest.cpp

```diff
@@ -1225,4 +1227,68 @@ TEST_F(IOGTest, testNetworkitBinaryZigzag) {
     }
 }
 
+TEST_F(IOGTest, testMatrixMarketReader) {
```

Comment:
Better have seperate tests for all cases (faster to track down if there are errors).